app/templates/header.html

```diff
@@ -23,7 +23,7 @@
             </a>
         </div>
         <div class="govuk-header__content">
-            <span class="app-service-name govuk-header__link--service-name">{{ service_title }}</span>
+            <span class="app-service-name govuk-header__link--service-name">{{ "Access Funding" if is_error else service_title }}</span>
```

Comment:
Even if that worked, would it not be awfully confusing to troubleshoot or decipher? As `service_title` could come from two different places. Not saying that `is_error` is the prettiest solution either.

Reply:
Potentially confusing, I guess that's fair, yeah.

Reply:
👍 I was thinking this but it is how a lot of the existing app is set up.

Having an individual handler override the value wouldn't give you an indication looking at the templates that it could ever be anything other than `service_title` which refers to the individual funds elsewhere.

I don't feel like there are enough shared requirements between the main app and these extra pages to try and fit them all into one -- I've previously just used different base templates to inherit from (i.e "base/logged in", "base/error", "base/logged out") but that benefits from generic components that can be used in multiple base templates and easily overriden. Looking at things like the header macro it looks there's a lot of delicate logic going on in there and I don't think its worth abstracting it out for this change.

Reply:
That's a lot of words just to avoid saying that you agree with my solution.

`service_title` was overwritten anyway by one of the `view` functions. I did an ugly `or` now to echo the view context before getting the `service_title` from the `context_processor`.